test/testcafe/spec/v1/PrimaryAuthForm_spec.js

```diff
@@ -117,7 +129,7 @@ test.requestHooks(logger)('Toggles icon when the password toggle button with fea
   await t.expect(primaryAuthForm.getInputField('password').getAttribute('type')).eql('text');
 });
 
-test.requestHooks(logger)('show anti-phishing message when security image is new user', async (t) => {
+test.requestHooks(logger, userImageMock)('show anti-phishing message when security image is new user', async (t) => {
```

Comment:
Missing request mock.
Test was working before because it used mock server from playground, see DISABLE_MOCK_SERVER